src/main/java/com/excella/foodinc/controller/FoodController.java

```diff
@@ -0,0 +1,21 @@
+package com.excella.foodinc.controller;
+
+import com.excella.foodinc.object.Food;
+import com.excella.foodinc.service.FoodService;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RestController
+public class FoodController {
+
+    private FoodService foodService = new FoodService();
+
+    @RequestMapping(value="/nutrition")
+    public Food retrieveFood(@RequestParam(value="name", required=false, defaultValue="pho") String name) {
+        foodService.Initialize();

```

Comment:
Look up Spring "Autowiring" and see how you don't need to ever initialize a dependency manually like you are .doing here.